caravel/static/default.css

```diff
@@ -59,7 +59,7 @@ body {
     left: 15px;
     right: 15px;
     padding: 5px 10px;
-    font: bold 1.5em/30px Roboto,sans-serif;
+    font: 300 1.5em/30px Roboto,sans-serif;
```

Comment:
We did. I just wanted to see how it looks with a thinner font. Your thoughts on the overall design?